packages/i18n/src/parser/utils/position.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Position in a text document expressed as zero-based line and character offset.
+ * The offsets are based on a UTF-16 string representation. So a string of the form
+ * `a𐐀b` the character offset of the character `a` is 0, the character offset of `𐐀`
+ * is 1 and the character offset of b is 3 since `𐐀` is represented using two code
+ * units in UTF-16.
+ *
+ * Positions are line end character agnostic. So you can not specify a position that
+ * denotes `\r|\n` or `\n|` where `|` represents the character offset.
+ *
+ * @description copied from: node_modules\vscode-languageserver-types\lib\umd\main.d.ts. To keep source code lightweight since it depends only on `Range`
+ */
+export interface Position {
```

Comment:
We should probably just add the dependency to `vscode-languageserver-types` like in `@sap-ux/odata-annotation-core-types`.

Reply:
> We should probably just add the dependency to `vscode-languageserver-types` like in `@sap-ux/odata-annotation-core-types`.

@voicis 
you mean `devDependency`?

Reply:
No actual dependency as we use the factory functions like `Position.create`